src/ServiceBusExplorer/Forms/MainForm.cs

```diff
@@ -1829,6 +1830,13 @@ private void RefreshIndividualTopic(TreeNode selectedNode)
                 return;
             }
 
+            //record the original subscriptions node is expanded
+            var originalSubscriptionsNode = selectedNode.Nodes.Find(SubscriptionEntities, false).First();
```

Comment:
If there's nothing, `.First()` will throw.

```suggestion
            var originalSubscriptionsNode = selectedNode.Nodes.Find(SubscriptionEntities, false).FirstOrDefault();
```

Reply:
yes, of course, my fault